### PR TITLE
refactor: manual redesign

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

**Comment by ije** [comment]
1. can we remove sidebar scroll?
2. i update the prev/next links style (please check figma)

**Comment by crowlKats** [comment]
> can we remove sidebar scroll?

but then how do we handle the sidebar being too long? in the figma you said to keep it the same

**Comment by ije** [comment]
what if we can collapse the menu?

Screen Shot 2022-07-26 at 17 01 23

**Comment by crowlKats** [comment]
sure, but we prob still should have scroll

File: components/Icons.tsx [hunk 1]
```diff
@@ -153,14 +153,18 @@ export function Cross(props: { class?: string }) {
...
+      viewBox="0 0 14 14"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
     >
       <path
-        fillRule="evenodd"
-        d="M12.707 5.293a1 1 0 010 1.414L9.414 10l3.293 3.293a1 1 0 01-1.414 1.414l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 0z"
-        clipRule="evenodd"
+        d="M9.5 2L4.5 7L9.5 12"
+        stroke="#232323"
```
Comment (ije) [inline components/Icons.tsx:164]: currentColor?

Comment (crowlKats) [reply components/Icons.tsx:164]: yea, will do

**Comment by ry** [comment]
Looks great!

**Comment by crowlKats** [comment]
@ije reworked the sidepanel. if it is fine how it is now, i will work on fixing the mobile view

**Review by ije — APPROVED** [review]
LGTM! nice job 👍

**Comment by kidonng** [comment]
Something seems to be off:

image

**Comment by crowlKats** [comment]
wow thats weird, will address ASAP

**Pull request merged**